cleanlab_studio/studio/trustworthy_llm.py

```diff
@@ -0,0 +1,53 @@
+from typing import cast, Literal, TypedDict
+from cleanlab_studio.internal.api import api
+
+
+QualityPreset = Literal["best", "high", "medium", "low", "base"]
+
+
+class TlmResponse(TypedDict):
```

Comment:
`TLMResponse`? For consistency with `TLM`

Reply:
OK w/ making both `Tlm`? That's the more pythonic naming scheme

Reply:
made `TLM` everywhere